### DISCUSSION TITLE
Where could I find the definition of the calculation process with dndlogdp?

### DISCUSSION THREAD

**Discussion by persisli** [quant-aq/py-smps#61] [Q&A] [answered]
Where could I find the definition of the calculation process with dndlogdp?Or could you tell me the specific definition file of smps function?

**Comment by dhhagan** [comment, marked as answer]
Hi @persisli! So sorry for the long delay in response here - I need to figure out how to better navigate GitHub notifications. The calculation process for `dndlogdp` can be found [here](https://github.com/quant-aq/py-smps/blob/master/smps/models.py#L144). Is there something specific you're looking for?
